stress/common/src/main/java/alluxio/stress/client/ClientIOTaskResult.java

```diff
@@ -202,8 +276,21 @@ public ClientIOTaskResult aggregate(Iterable<TaskResult> results) throws Excepti
               String series = summary.mParameters.getDescription(fieldNames.getSecond());
               responseTimeGraph.addDataSeries(series, summary.getThroughputData());
               responseTimeGraph.setErrors(series, summary.collectErrors());
-            }
 
+              for (Map.Entry<String, SummaryStatistics> entry :
+                  getStatisticsPerMethod().entrySet()) {
+                final String method = entry.getKey();
+                final LineGraph.Data timeToFirstByte = entry.getValue().computeTimeData();
+
+                if (method.equals("readChunk")) {
+                  LineGraph timeToFirstByteLineGraph =
+                      new LineGraph(operation + " - Time To First Byte (ms) " + method, subTitle,
+                          "Percentile", "Time To First Byte (ms)");
+                  timeToFirstByteLineGraph.addDataSeries(series, timeToFirstByte);
+                  graphs.add(timeToFirstByteLineGraph);
+                }
+              }
+            }
             graphs.add(responseTimeGraph);
```

Comment:
Can we add this response time graph before the time to first byte graph? You can just move this line up before the for loop, since it will add it to the list first.